### DISCUSSION TITLE
Golang Sessions vs Python

### DISCUSSION THREAD

**Discussion by mustafaaljadery** [confetti-framework/confetti#119] [Q&A]
Let's say I have these two functions:

https://play.golang.org/p/mh3tDel8VU7

What I am trying to do is pass a session variable inside of the function parameters. This is how it is done in python. I'm wondering how I can do it in golang:

https://www.codepile.net/pile/P7eXDapn

The whole point is that I run the same session when I run both functions. One after the other.

**Comment by reindert-vetter** [comment]
@mustafaaljadery Are you running on 1 server or on several?
